mlflow_project/src/rf_test/train.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+from functools import partial
+
+import mlflow
+import mlflow.sklearn
+
+from cuml.metrics.accuracy import accuracy_score as accuracy_score
+from cuml.preprocessing.model_selection import train_test_split as train_test_split
+from cuml.ensemble import RandomForestClassifier as RandomForestClassifier
+
+from hyperopt import fmin, tpe, hp, Trials, STATUS_OK
+
+
+def load_data(fpath):
+    """
+    Simple helper function for loading data to be used by GPU models.
+
+    :param fpath: Path to the data to be ingested
+    :return: DataFrame wrapping the data at [fpath]. Data will be in either a Pandas or RAPIDS (cuDF) DataFrame
+    """
+    import cudf
+
+    df = cudf.read_parquet(fpath)
```

Comment:
Thank you for the simplification to just GPU! Maybe we should have load_data return X_train, X_test, y_train, y_test instead of data now that it's so simple?
(I suggest we defer this to a future PR though so we get the basics in)